### PR TITLE
[Dev] Adding realistic output names for stubs

### LINKED ISSUE #13: Update the stubs for realistic file names based on the mock files
We should update the filenames generated by `stub` to be more realistic, based on a previous run of `mtbseq` shared with @Mxrcon earlier.

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey, I've fixes the stubs outputs aiming to fix #13 with very simple changes.

This pr only contains new outputs names for the stubs developed before.

I'll e happy on hearing you opinion, feel free to discuss it here! 
I'm at your disposal, Hope we have a merge!
Warmly, Davi

### CODE REVIEW

File: modules/mtbseq/tbgroups/tbgroups.nf [hunk 1]
```diff
@@ -35,8 +35,8 @@ process TBGROUPS {
     sleep \$[ ( \$RANDOM % 10 )  + 1 ]s
 
     mkdir Groups
-    touch Groups/${params.mtbseq_project_name}_joint_[mincovf]_[mincovr]_[minfreq]_[minphred20]_samples_amended_[unambig]_phylo_[window].matrix
-    touch Groups/${params.mtbseq_project_name}_joint_[mincovf]_[mincovr]_[minfreq]_[minphred20]_samples_amended_[unambig]_phylo_[window]_[distance].groups
+    touch Groups/${params.mtbseq_project_name}_joint_cf4_cr4_fr75_ph4_samples6_amended_u95_phylo_w12.matrix
```
Comment (abhi18av) [inline modules/mtbseq/tbgroups/tbgroups.nf:38]: Perhaps we should use `samples5` as defined in the `mock_data` folder 

https://github.com/emilyncosta/mtbseq-nf/tree/dev/data/mock_data

**Review by abhi18av — CHANGES_REQUESTED** [review]
Hi @Mxrcon , 

Thanks for the PR, we are pretty close already. Just one change overall, mentioned below.

Also, I noticed that the mock data https://github.com/emilyncosta/mtbseq-nf/tree/dev/data/mock_data  `AD04` and `AD05` are incomplete. 

I'd suggest that we keep only 5 samples overall in the mock and remove `AD06` after fixing `AD04/5`

File: modules/mtbseq/tbgroups/tbgroups.nf (hunk 1 above)
Comment (abhi18av) [reply modules/mtbseq/tbgroups/tbgroups.nf:38]: Therefore `sample5` would be used throughout the stubs in other processes as well.

**Review by abhi18av — APPROVED** [review]
Yup, this looks good to merge now ✅ 

Thanks @Mxrcon!

**Comment by Mxrcon** [comment]
Thank you Abhinav, i forgot to ask a new review :eyes:

**Pull request merged**